manager/integration/tests/test_node.py

```diff
@@ -2184,3 +2184,49 @@ def test_replica_scheduler_rebuild_restore_is_too_big(client):  # NOQA
     node = common.wait_for_disk_update(client, lht_hostId,
                                        len(update_disks))
     cleanup_host_disk(client, 'vol-small')
+
+
+@pytest.mark.skip(reason="TODO")
+def test_node_eviction():
+    """
+    Test node eviction (assuming this is a 3 nodes cluster)
+
+    1. Disable scheduling on node 3.
+    2. Create volume 1 with 2 replicas.
+    3. Attach volume 1 to node 1 and write some data to it and get
+    checksum 1.
+    4. Set 'Eviction Requested' to 'true' and disable scheduling on node 1.
```

Comment:
Shouldn't be the disabling of scheduling set first and then eviction requested should be set to ```True``` ?

Reply:
I was thinking to set them at the same time. Let me put the disabling scheduling before the eviction to solve the confusion.